### ISSUE TITLE
testthat tests failing on Linux

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1146]
Issue #1145 has been focused an issue specific to Windows and file encodings, but when trying to test the package on Linux I noticed two other failures. Though it best to document in case it's different from things @dankelley is seeing:
```
Failed -------------------------------------------------------------------------
1. Failure: a bioargo dataset (@test_local_argo.R#41) --------------------------
sort(names(a@data)) not equal to c(...).
4/22 mismatches
x[1]: "BBP700"
y[1]: "bbp700"

x[2]: "BBP700Adjusted"
y[2]: "bbp700Adjusted"

x[3]: "BBP700AdjustedError"
y[3]: "bbp700AdjustedError"

x[19]: "temperatureOxygen"
y[19]: "temperatureeratureOxygen"


2. Failure: woce 2 (@test_local_ctd.R#29) --------------------------------------
woce[["conductivityUnit"]] not equal to list(unit = expression(), scale = "").
target is NULL, current is list
```

**Comment by dankelley** [comment]
Thanks -- I see why those occur.  (Damned if I know why they don't occur for me on osx.)

I've also got what may be a solution to the other issue #1145 and will be submitting to winbuilder since I've got it narrowed down well enough to only (I think) require two test runs.

**Comment by dankelley** [comment]
Oh, I wasn't testing them the right way. For the record:

1. Testing with "R CMD check" does not work since the requisite data files are not saved in the tarball.
2. Testing in Rstudio by loading the `test_local.R` *only* works if you first `setwd("tests/testthat")`. (this was what I was not seeing.)

**Comment by dankelley** [comment]
Possibly fixed in `develop` commit f8df66d5af35c2255d5fdba27c5518d84285ec77 (although that still has the `read.ctd.sbe` debugging lines mentioned in #1145 )

**Comment by richardsc** [comment]
Great! This seems fixed now. Closing.
